scene/gui/file_dialog.cpp

```diff
@@ -1470,6 +1530,7 @@ FileDialog::FileDialog() {
 	tree->connect("item_activated", callable_mp(this, &FileDialog::_tree_item_activated));
 	tree->connect("nothing_selected", callable_mp(this, &FileDialog::deselect_all));
 	dir->connect("text_submitted", callable_mp(this, &FileDialog::_dir_submitted));
+	filename_filter->connect("text_changed", callable_mp(this, &FileDialog::_filename_filter_changed));
```

Comment:
```suggestion
	filename_filter->connect("text_changed", callable_mp(this, &FileDialog::_filename_filter_changed).unbind(1));
```
Instead of just ignoring the argument